ui/src/app/flags/analytics/Analytics.tsx

```diff
@@ -61,26 +62,33 @@ const durations: FilterableDurations[] = [
 export default function Analytics() {
   const [selectedDuration, setSelectedDuration] =
     useState<FilterableDurations | null>(durations[0]);
+  const [pollingInterval, setPollingInterval] = useState<number>(0);
+
   const { flag } = useOutletContext<AnalyticsProps>();
   const namespace = useSelector(selectCurrentNamespace);
 
   const nowISO = parseISO(new Date().toISOString());
 
-  const getFlagEvaluationCount = useGetFlagEvaluationCountQuery({
-    namespaceKey: namespace.key,
-    flagKey: flag.key,
-    from: format(
-      addMinutes(
+  const getFlagEvaluationCount = useGetFlagEvaluationCountQuery(
+    {
+      namespaceKey: namespace.key,
+      flagKey: flag.key,
+      from: format(
         addMinutes(
-          nowISO,
-          selectedDuration?.value ? selectedDuration.value * -1 : -60
+          addMinutes(
+            nowISO,
+            selectedDuration?.value ? selectedDuration.value * -1 : -60
+          ),
+          nowISO.getTimezoneOffset()
         ),
-        nowISO.getTimezoneOffset()
+        timeFormat
       ),
-      timeFormat
-    ),
-    to: format(addMinutes(nowISO, nowISO.getTimezoneOffset()), timeFormat)
-  });
+      to: format(addMinutes(nowISO, nowISO.getTimezoneOffset()), timeFormat)
+    },
+    {
+      pollingInterval
```

Comment:
how does this work?

Reply:
https://redux-toolkit.js.org/rtk-query/usage/polling

Reply:
@markphelps Yeah I was about to point to those docs as well. I followed the example there.

Reply:
nice!